Packs/Infoblox/Integrations/Infoblox/Infoblox.py

```diff
@@ -345,7 +475,7 @@ def get_ip_command(client: Client, args: Dict[str, str]) -> Tuple[str, Dict, Dic
     Returns:
         Outputs
     """
-    ip = args.get('ip')
+    ip = str(args.get('ip'))
```

Comment:
If the argument is mandatory, casting is unnecessary as it will always be part of args.
Otherwise, this will result in `ip = 'None'` - this is not what you want.
better to have a default value of an empty string